server/lobbyconnection.py

```diff
@@ -88,6 +117,36 @@ def __init__(
 
         self._logger.debug("LobbyConnection initialized")
 
+    class Decorators:
```

Comment:
It cannot be an instance method, since it is called without an instance, but it cannot be a static method either (see below). I didn't want to declare it outside the class, since it knows too much about `LobbyConnection` (although it's maybe unlikely that anything will ever inherit from `LobbyConnection`?).
An inner class is one of the workarounds recommended here:
https://medium.com/@vadimpushtaev/decorator-inside-python-class-1e74d23107f6